frontend/packages/ui/src/ProgressBar.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import { BarChart, Bar, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts'
+import { Colors } from './styles/colors'
+
+interface Props {
+  data: { name: string; size: number; color: string }[]
```

Comment:
We need to support data like this (percentage array):
![image](https://user-images.githubusercontent.com/22115870/142047962-d81af13d-0315-4975-bd25-73c0b4d7ac90.png)

and we need to handle colors inside the component (Ana will send us range of 10 colors).

Reply:
I used this two types of data as well. I was not sure I we need to support the data that was already in mock, so I left it.
If it is just these two types of data, I will remove options for the previous one.